index.html

```diff
@@ -21,9 +21,9 @@
         <img src="res/wanRight.svg" onerror="this.src='res/wanRight.png'" class="wan_text" id="wanRight">
         <div id="wanCenter"></div>
         <div id="footer-info">
-            <a href="#" onclick="wolfMomiji()">Momiji</a>
-            <a href="#" onclick="wolfHolo()">Holo</a>
-            <a href="#" onclick="wolfKaren()">Karen</a>
+            <a href="#momiji" onclick="wolfMomiji()">Momiji</a>
```

Comment:
Each a tag should have a hash value. This makes it easier to copy and paste a URL to send to others.